dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -109,3 +109,6 @@ sameSiteCookieDisabled=Do not set the SameSite attribute value on the SSO cookie
 sameSiteCookielax=Set the SSO cookie SameSite attribute value to Lax
 sameSiteCookieNone=Set the SSO cookie SameSite attribute value to None. When None is set, the secure attribute is set to true on the cookie. 
 sameSiteCookieStrict=Set the SSO cookie SameSite attribute value to Strict
+
+loggedOutCookieCacheJCacheRef=JCache reference
+loggedOutCookieCacheJCacheRef.desc=The JCache reference that will be used as the logged out cookie cache.
```

Comment:
suggested edit

The JCache reference that is used as the logged-out-cookie cache.

Reply:
Updated.